### PR TITLE
Remove deprecated ledger fields

### PR DESCRIPTION
**Pull request by intelliot**
Per ripple/rippled#3214

The following fields are deprecated and may be removed without further notice: accepted, hash (use ledger_hash instead), seqNum (use ledger_index instead), totalCoins (use total_coins instead).

### CODE REVIEW

**Comment by FKSRipple** [comment]
Can you remove from relevant code / type definitions as well? Ex: I see `seqNum` used in:

- `src/offline/ledgerhash.ts`
- `src/ledger/parse/ledger.ts`
- `src/common/types/objects/ledger.ts`

**Comment by intelliot** [comment]
> Can you remove from relevant code / type definitions as well? Ex: I see `seqNum` used in:
> 
> * `src/offline/ledgerhash.ts`
> * `src/ledger/parse/ledger.ts`
> * `src/common/types/objects/ledger.ts`

Done

**Comment by intelliot** [comment]
This might technically be a breaking change, but these fields are directly redundant with the snake_case versions and have been deprecated since ~2015. So I think this would be fine to merge

File: src/ledger/parse/ledger.ts [hunk 1]
```diff
@@ -67,19 +66,19 @@ function parseState(state) {
  * @throws RangeError: Invalid time value (rippleTimeToISO8601)
  */
 export function parseLedger(ledger: Ledger): FormattedLedger {
-  const ledgerVersion = parseInt(ledger.ledger_index || ledger.seqNum, 10)
```
Comment (natenichols) [inline src/ledger/parse/ledger.ts:70]: Any particular reason why we're using `parseInt(..., 10)`, could we just use Number(...) if parsing in base 10?

Comment (intelliot) [reply src/ledger/parse/ledger.ts:70]: No reason. I suspect `Number()` would work equally well, but I'm not 100% sure. It has been `parseInt()` from [the beginning of time](https://github.com/ripple/ripple-lib/pull/504/files#diff-4f915b23ac0d38c14331690bd88e1b4af5588dacf2f87fd658748bebb21742f1R39) and I think we can leave it that way unless there is some reason to change it.

Comment (natenichols) [reply src/ledger/parse/ledger.ts:70]: I'm perfectly happy with `parseInt()`, mostly just curious.

**Review by natenichols — APPROVED** [review]

**Pull request merged**